pytesseract/pytesseract.py

```diff
@@ -209,7 +218,14 @@ def save(image):
     try:
         with NamedTemporaryFile(prefix='tess_', delete=False) as f:
             if isinstance(image, str):
-                yield f.name, realpath(normpath(normcase(image)))
+                if image.startswith('http:') or image.startswith('https:'):
```

Comment:
Couldn't this be shorter using `image.startswith(prefix=('http:', 'https:'))`?

Reply:
Good point, fixed